config/rbac/namespaced/role_binding.yaml

```diff
@@ -9,5 +9,5 @@ roleRef:
   name: manager-role
 subjects:
 - kind: ServiceAccount
-  name: default
+  name: nexus-operator-sa
```

Comment:
this should be `name: sa` with no `namespace` (`kustomize` will add the correct namespace)